Dockerfile.tpu

```diff
@@ -8,6 +8,9 @@ WORKDIR /workspace
 RUN python3 -m pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
 RUN python3 -m pip install torch_xla[pallas] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html
 
+# for testing torch.compile
+RUN python3 -m pip install git+https://github.com/thuml/depyf.git
+
```

Comment:
While I do understand that depyf can be a good for debugging, I don't want to include it in `Docker.tpu` since it can also be used for actual deployment.

Reply:
moved to test script only. does this sound good to you?